packages/components/src/Filter/FilterBar/subcomponents/AddFiltersMenu/AddFiltersMenu.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react"
 import { useIntl } from "@cultureamp/i18n-react-intl"
-import { Button } from "~components/Button"
 import { AddIcon } from "~components/Icon"
-import { Menu, MenuList, MenuItem } from "~components/Menu"
+import { Menu, MenuList, MenuItem } from "~components/__actions__/v1"
```

Comment:
Why not v2?

Reply:
This would be part of the Menu PR that's brought across since I needed to branch of Douglet's work 😉 but valid comment, cc: @dougmacknz

Reply:
Updated now on my branch

Reply:
Noice ✋ 5️⃣  I'll correct all those when I rebase off main